docs/user/types/references/reference/summary-usage.en.md

```diff
@@ -7,6 +7,9 @@ For other documents it's the document's title.
 
 For different versions of Advisories it has the following format:
```

Comment:
Advisories => advisories
I guess it is more about the formats than versions.

Reply:
changed to mention formats of advisories and adjusted a little